examples/window-app/nrfconnect/main/include/CHIPProjectConfig.h

```diff
@@ -31,3 +31,8 @@
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR 0xF00
 
 #define CHIP_DEVICE_CONFIG_SED_SLOW_POLLING_INTERVAL 2000_ms32
```

Comment:
I guess this is obsolete now. Rename it to `CHIP_DEVICE_CONFIG_SED_SLOW_POLLING_INTERVAL`, too?

Reply:
Fixed by rebase.